src/plugin/leaflet.canvas-markers.js

```diff
@@ -3,6 +3,13 @@
 function layerFactory(L) {
 
     var CanvasIconLayer = (L.Layer ? L.Layer : L.Class).extend({
+        options: {
+            // @option padding: Number = 0.1
+            // How much to extend the clip area around the map view (relative to its size)
+            // e.g. 0.1 would be 10% of map view in each direction
+            padding: 0.1
+        },
+
```

Comment:
Is there similar options in Leaflet?

Reply:
In Leaflet `padding` value is specified in `Renderer` class (`var Renderer = Layer.extend({`) and is not overridden.
It would be useful to assign an indent equal to that in `window.map` in iitc, but this is hardly possible.

Reply:
> It would be useful to assign an indent equal to that in window.map in iitc, but this is hardly possible.

Why hardly? I suppose we just need `map._renderer.options.padding`.
Or `map.getRenderer({options:{}}).options.padding`.